### DISCUSSION TITLE
large number of prior knowledge from cmd line

### DISCUSSION THREAD

**Discussion by CJSmithing** [asreview/asreview#1903] [Q&A] [answered]
Hi! I want to run a simulation from command line where I have over 3000 specific references for my prior knowledge. As of what I know now, I will have to type out each reference's index (0 1 2 3 4 5 6 ... and so on) which is too long a line for my command window to run. Is there a way other than typing out each references index for --prior_idx to make this work?

**Comment by Rensvandeschoot** [comment, marked as answer]
You will enjoy  [Makita](https://www.softwareimpacts.com/article/S2665-9638(24)00051-4/fulltext) (or on [Github](https://github.com/asreview/asreview-makita//)); a workflow generator for preparing the scripts for large simulation studies :-)
One of the templates (i.e., [prior](https://github.com/asreview/asreview-makita//?tab=readme-ov-file#prior-template)) has recently been added for exactly your purpose. 

@jteijema what is the status of the two open pull requests??

**Reply by CJSmithing** [reply]
Thank you! This was exactly what I was looking for.

**Reply by jteijema** [reply]
@Rensvandeschoot The last PR will be approved today, and the prior template is ready for use!
